src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx

```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
     );
     expect(tree.props.children).to.be.null;
   });
-
-  it('should hide requested files when closed', () => {
-    const claim = {
-      attributes: {
-        open: false,
-        eventsTimeline: [],
-      },
-    };
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-
-    expect(tree.subTree('RequestedFilesInfo')).to.be.false;
-  });
-
-  it('should show requested files when open', () => {
-    const claim = {
-      attributes: {
-        open: true,
-        eventsTimeline: [],
-      },
-    };
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.subTree('RequestedFilesInfo')).not.to.be.false;
-  });
-
-  it('should render ask va to decide component', () => {
-    const claim = {
-      id: 1,
-      attributes: {
-        phase: 3,
-        documentsNeeded: false,
-        decisionLetterSent: false,
-        waiverSubmitted: false,
-        eventsTimeline: [
-          {
-            type: 'still_need_from_you_list',
-            status: 'NEEDED',
-          },
-        ],
-      },
-    };
-
-    const tree = SkinDeep.shallowRender(
-      <FilesPage params={{ id: 2 }} claim={claim} />,
-    );
-    const content = tree.dive(['FilesPageContent']);
-    expect(content.everySubTree('AskVAToDecide')).not.to.be.empty;
-  });
-
-  it('should display turned in docs', () => {
```

Comment:
I think we want this test we just need to update the claim for lighthouse

Reply:
Done